src/Metrics.h

```diff
@@ -259,6 +259,16 @@ class TopN final : public Metric
         _fi.merge(other._fi);
     }
 
+    void update_top_count(const size_t top_count)
```

Comment:
the convention is to prepend `set_` for setters, and for accessors to have the same name as the variable without the prepended underscore, so these should be `set_top_count` and `top_count`

Reply:
done